src/js/structs/ServiceTree.js

```diff
@@ -48,6 +48,9 @@ module.exports = class ServiceTree extends Tree {
 
       // Check the DCOS_PACKAGE_FRAMEWORK_NAME label to determine if the item
       // should be converted to an Application or Framework instance.
+      if (item instanceof Framework || item instanceof Application) {
```

Comment:
Let's add this checks to the condition in line 37 https://github.com/dcos/dcos-ui/pull/903/files#diff-0eb72bc636980e431e5fe14b2a54616dR37